src/components/scrollManager.js

```diff
@@ -372,7 +372,7 @@ define(["dom", "browser", "layoutManager"], function (dom, browser, layoutManage
      */
     function useSmoothScroll() {
 
-        if (browser.tizen) {
+        if (browser.tizen || browser.web0s || browser.firefox || browser.chrome) {
```

Comment:
I know lots of people don't like smooth scroll, so we can't just enable it everywhere by default.